clusterloader2/testing/load/experimental-config.yaml

```diff
@@ -113,7 +113,7 @@ steps:
     Method: APIResponsivenessPrometheus
     Params:
       action: start
-  - Identifier: LoadPodStartupLatency
+  - Identifier: PodStartupLatency
```

Comment:
There is another PodStartupLatency measurement already defined in this config, line 481.
I consciously named it differently to avoid conflicts and to be able to differentiate them.

Reply:
Changed name to `Load...` to `Second...` so i's not confusing from perf-dash perspective how metric name is generated.